distribution/ddf-common/src/main/resources/bin/setenv-wrapper.conf

```diff
@@ -0,0 +1,67 @@
+# Log format to include log level, prefix, milliseconds, and the message
+wrapper.console.format=LPZM
+
+# Enables restart of JVM when it exits with 10. 10 is normally used by the karaf script to
+# indicate a restart
+wrapper.on_exit.10 = RESTART
+
+# Defines a command file to be used by the ddf_on_error.sh script when executed following
+# a non-recoverable system error and the JVM should be restarted
+wrapper.commandfile = %KARAF_HOME%/bin/restart.jvm
+
+# Additional JVM parameters
+# note that n is the parameter number starting from 11 (1-10 is defined in ddf-wrapper.conf).
+# make sure there is no gap
+wrapper.java.additional.11 = -Dddf.home=%KARAF_HOME%
```

Comment:
❓ Windows-style variable interpolation? This shouldn't be `$KARAF_HOME` or `${KARAF_HOME}`?

Reply:
This is a service wrapper configuration and not a bash or command. These are expanded by the wrapper.